app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -85,17 +98,20 @@ class OptionsContainer extends React.Component {
       answer: this.props.answer,
       onChange: this.onChange.bind(this),
       onSave: this.onSave.bind(this),
+      onFileUpload: this.onFileUpload.bind(this),
+      onFileDelete: this.onFileDelete.bind(this),
+      onFileDownload: this.onFileDownload.bind(this),
       onEnter: () => this.setState({ selected: true }),
       onLeave: () => this.setState({ selected: false }),
-      focusOnMount: this.props.focusOnMount,
+      focusOnMount: this.props.focusOnMount
```

Comment:
Please keep this - it'll help our diffs cleaner if/when someone adds keys and prepares us for when I drop a linter with Airbnb JS rules on this thing ha ha